tests/README.md

```diff
@@ -2,27 +2,47 @@
 
 ## Overview
 
-This project contains UI smoke tests for Busola. Copy your cluster's kubeconfig file to [`fixtures/kubeconfig.yaml`](fixtures/kubeconfig.yaml).
+This project contains smoke and integration UI tests for Busola. Copy your cluster's kubeconfig file to [`fixtures/kubeconfig.yaml`](fixtures/kubeconfig.yaml).
 
 ## Installation
 
 To install dependencies, run the `npm install` command.
 
 ## Development
 
-To run UI tests inside a container in the cluster using a Chrome browser, use this command:
+To run Cypress UI tests using a Chrome browser in headless mode,
+pointing to remote Busola cluster with default local.kyma.dev domain use this command:
```

Comment:
Maybe we can highlight important phrases with code or bold font or something. For example:


To run Cypress UI tests using a Chrome browser in `headless mode`,
pointing to `remote Busola` cluster with default `local.kyma.dev` domain use this command:

or

To run Cypress UI tests using a Chrome browser in **headless mode**,
pointing to **remote Busola** cluster with default `local.kyma.dev` domain use this command:

Reply:
Apparently we don't use **bold** anywhere in readme so use `code`